packages/core/src/node/messaging/messaging-backend-module.ts

```diff
@@ -34,4 +35,7 @@ export const messagingBackendModule = new ContainerModule(bind => {
     bind(BackendApplicationContribution).toService(MessagingContribution);
     bind(MessagingListener).toSelf().inSingletonScope();
     bindContributionProvider(bind, MessagingListenerContribution);
+    bind(HttpWebsocketAdapter).toSelf().inRequestScope();
```

Comment:
Just a question: why `inRequestScope()` here? Do you anticipate more than one `@inject`ion of this class during a single `.get()` for a given container?